datacats/userprofile.py

```diff
@@ -131,7 +131,6 @@ def create(self, environment, target_name, stream_output=None):
         remote_server_command(
             ["ssh", environment.deploy_target, "create", target_name],
             environment, self,
-            stream_output=stream_output,
```

Comment:
Should I? I guess I may do that. There is no rush though, maybe we will need it for testing or something